### PR TITLE
PCL SiPixel Alignment: integration of quality checks on payload and DQM module

### PR DESCRIPTION
**Pull request by cerminar**
This PR modifies only the PCL sequences for the alignment of the Pixel large structures.
It integrates the needed checks for the creation of the payload and the DQM module to monitor the movements of the alignables.

This is needed to start testing the new workflow in production environment.

This is equivalent to PR #13983 for 81X.

### CODE REVIEW

**Comment by mmusich** [comment]
please test

File: Alignment/MillePedeAlignmentAlgorithm/interface/MillePedeFileReader.h [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+        int alignable      = std::stoi(tokens[0]);
+        int alignableIndex = alignable % 10 - 1;
+
+        double ObsMove = std::stof(tokens[3]) * Multiplier[alignableIndex];
+        double ObsErr  = std::stof(tokens[4]) * Multiplier[alignableIndex];
+
+        int det = -1;
+
+        if (alignable >= 60 && alignable <= 69) {
```
Comment (mmusich) [inline Alignment/MillePedeAlignmentAlgorithm/interface/MillePedeFileReader.h]: @ghellwig for the sake of curiosity how is the millepede internal labeling going to behave with the phase-I geometry?

Comment (ghellwig) [inline Alignment/MillePedeAlignmentAlgorithm/interface/MillePedeFileReader.h]: I think, as long as the number of alignables does not exceed 700000 we are fine:

https://github.com/cerminar/cmssw/blob/PCLDQMPlots_80X_v2/Alignment/MillePedeAlignmentAlgorithm/src/PedeLabelerBase.cc#L22

Haven't checked it, though.

**Comment by cerminar** [comment]
Ops, this PR was obviously meant for 80X since there is already one in 81X, sorry for the noise.

**Pull request closed without merging**